package.json

```diff
@@ -4,9 +4,6 @@
   "description": "Universal front-end projects to complement addons-server.",
   "main": "index.js",
   "private": true,
-  "engines": {
-    "node": ">=6 <=8"
```

Comment:
Why did you remove it entirely? I think it would be nice (for contributors) to give them an error when they aren't on node 8. This should produce an error for them:

```
    "node": ">=8.10"
```